mods/ra/maps/soviet-01/rules.yaml

```diff
@@ -13,6 +13,12 @@ V01:
 	SpawnActorOnDeath:
 		Actor: healcrate
 
+powerproxy.paratroopers:
+	ParatroopersPower:
+		DisplayBeacon: False
+		DropItems: E1,E1,E1,E2,E2
+	AlwaysVisible:
```

Comment:
This should not be needed.

Reply:
> pchote: @Smitty I think we need that to prevent linter failures
> pchote: make sure it’s not that before you break your prs!

For whatever reason when I updated this pr without `AlwaysVisible` Travis didn't catch anything. I then updated the rest of my missions that have these paradrops and had to undo all that after Travis got upset.

Reply:
That is really odd, what did Travis report? Removing this line locally and running the linter didn't give me any errors.

Reply:
I forget but removing the lines from missions like #18098 caused an error. Could it be that `powerproxy.paratroopers:` is understood without the line but a custom name like `INSERTIONDROP:` will cause a problem? If so I could remove the line here but leave it in the other PRs.

Reply:
Yes, that will be the difference since `powerproxy.paratroopers` has predefined values at: https://github.com/OpenRA/OpenRA/blob/3f5fadf2e9897973311cc0dbdfc3fde036644ff3/mods/ra/rules/misc.yaml#L345